dockstore-launcher/src/main/java/io/dockstore/common/FileProvisionUtil.java

```diff
@@ -109,16 +108,13 @@ public void bytesTransferred(long totalBytesTransferred, int bytesTransferred, l
                 printer.handleProgress(totalBytesTransferred, streamSize);
             }
         };
-        try (OutputStream outputStream = outputSteam) {
+        try {
             // a larger buffer improves copy performance
             // we can also split this (local file copy) out into a plugin later
             final int largeBuffer = 100;
             Util.copyStream(inputStream, outputStream, Util.DEFAULT_COPY_BUFFER_SIZE * largeBuffer, inputSize, listener);
         } catch (IOException e) {
             throw new RuntimeException("Could not provision input files", e);
-        } finally {
-            IOUtils.closeQuietly(inputStream);
-            System.out.println();
```

Comment:
This seems suspicious (albeit ugly). This looks like it might have been used to add new lines after the progress bar or similar.